Add Made in India collection

mvkaran opened the pull request:
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

<!-- ⚠️ ... or this section ⚠️ -->
### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Many developers in India have been asking about which are the projects that were created in India and/or made in India. This collection curates some of the projects that can help Indian devs discover them.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

gallexi reviewed (approved):
This is so awesome, thank you for this contribution!!

I wonder if we could find a good picture to use as an icon for this collection, like the Indian flag or something! 🤔  

Either way, this is good as-is too. I'll approve it now, let me know if you would like to add a picture 🇮🇳  ✨

mvkaran commented:
@gallexi Good idea! I've added a picture of the flag and also added some more items to the collection.

Source of image (official): https://knowindia.gov.in/national-identity-elements/national-flag.php

gallexi commented:
Excellent, thanks!!
